src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -152,6 +155,22 @@ public function iFilterByProduct(string ...$productNames): void
         $this->client->filter();
     }
 
+    /**
+     * @When I resend the shipment confirmation email
+     */
+    public function iResendTheShipmentConfirmationEmail(): void
+    {
+        /** @var RequestInterface $request */
+        $request = $this->requestFactory->custom(
+            \sprintf('%s/admin/shipments/resend-shipment-confirmation-email', $this->apiUrlPrefix),
+            HttpRequest::METHOD_POST,
+        );
+
+        $request->setContent(['shipmentId' => $this->sharedStorage->get('order')->getShipments()->first()->getId()]);
```

Comment:
Same as with payments collection, actual/newer attempts are placed later, though this system is implied, but not used at standard sylius.

```suggestion
        $request->setContent(['shipmentId' => $this->sharedStorage->get('order')->getShipments()->last()->getId()]);
```

On separate note, is it necessary to assert, that values are available?